Update user buttons

EngrTaofeek opened the pull request:
fixes #740    Fixed border corner radius and allowed depth button outline show above the image. And also background. fixes #726 
![WhatsApp Image 2022-01-26 at 00 39 27](https://user-images.githubusercontent.com/38661872/151078099-21abcf52-52ba-422d-bcf4-e7d00f29746c.jpeg)

EngrTaofeek commented:
@Elforama I had a challenge implementing the shadow effect for a selected grower button as shown here https://www.sketch.com/s/a0bbeac0-767a-4873-beef-c6c7ffd5826a/a/4a55aKo  The best way to create this effect is to use radial gradients. However radial gradients don't quite work well with other shapes except if they are circular or oval, see what it looks like(it takes the oval shape).
![WhatsApp Image 2022-01-26 at 10 23 04](https://user-images.githubusercontent.com/38661872/151139076-10e73fee-2b22-42f8-9023-f1972f869acd.jpeg)
Other available options were to use another external library(https://github.com/PatilSiddhesh/Holi) or spend more time trying to create the effect from the scratch.
I explored with other gradients that works perfectly. Just in case you want me to use another one instead of spending more time on that
![WhatsApp Image 2022-01-26 at 10 23 48](https://user-images.githubusercontent.com/38661872/151139895-4ba57d92-b690-4ff5-a39b-cb25ee3771a8.jpeg)
![WhatsApp Image 2022-01-26 at 10 24 22](https://user-images.githubusercontent.com/38661872/151139901-de06b1eb-36f7-4c00-8866-a7ce76340fc1.jpeg)
![WhatsApp Image 2022-01-26 at 10 25 08](https://user-images.githubusercontent.com/38661872/151139905-23c3f798-28df-42db-983c-5adfd21ff597.jpeg)
![WhatsApp Image 2022-01-26 at 10 25 46](https://user-images.githubusercontent.com/38661872/151139909-2e13fc72-4da9-43db-aa65-7211f5e451aa.jpeg)
![WhatsApp Image 2022-01-26 at 10 26 58](https://user-images.githubusercontent.com/38661872/151139911-bfb6f3f6-a776-4e47-b59a-2e7445fa82b4.jpeg)
![WhatsApp Image 2022-01-26 at 10 28 08](https://user-images.githubusercontent.com/38661872/151139915-a5f2f9cb-0224-4ea7-be17-d4d1a9ccff53.jpeg)

@Elforama and @snacks what do you think?

Elforama commented on app/src/main/java/org/greenstand/android/TreeTracker/view/SelectableItemDetail.kt:
```diff
@@ -62,7 +53,7 @@ fun SelectableImageDetail(
                     .wrapContentHeight()
                     .aspectRatio(1.0f)
-                    .clip(RoundedCornerShape(percent = 10)),
+                    .clip(RoundedCornerShape( 10.dp)),
```
<nit> please don't add extra spacing for params

Elforama reviewed (changes requested):
I think the modifications to this prior to this PR where to add a green outline when selected. I think we can make this a default button without any glow and 'if' statements. It can work much like the language selection screen.

Once that is cleaned up we can then make a 'glow' modifier. Something that way we just add a glow and the whatever view we want can start to glow.

EngrTaofeek commented:
> I think the modifications to this prior to this PR where to add a green outline when selected. I think we can make this a default button without any glow and 'if' statements. It can work much like the language selection screen.
> 
> Once that is cleaned up we can then make a 'glow' modifier. Something that way we just add a glow and the whatever view we want can start to glow.

Okay great. I haven't added that green outline feature to this branch yet. I have fixed the extra spacing you pointed out and rebased the branch.

Elforama commented:
Can you please attach a new screenshot that shows the current state?

EngrTaofeek commented:
@Elforama  it looks like this now (default and selected state)
![WhatsApp Image 2022-02-02 at 00 19 54](https://user-images.githubusercontent.com/38661872/152068077-5de9db10-88a2-4be4-84fc-223bf1755a67.jpeg)

![WhatsApp Image 2022-02-02 at 00 19 21](https://user-images.githubusercontent.com/38661872/152068073-af155ae1-3954-4228-8ac3-f9ceb26c3b68.jpeg)
